README.md

```diff
@@ -36,15 +36,25 @@ os.system("cat periodic_torsion_k_constants_fit_energy.txt")
     - "opls_all_summed_dihedrals_k_constants_figure.pdf"
 
 
-## Installation/Setup
-```
-conda install -c conda-forge mosdef-dihedral-fit
-git clone https://github.com/GOMC-WSU/GOMC.git
+## Quick Installation/Setup
+
+```bash
+mamba install -c conda-forge mosdef-dihedral-fit
```

Comment:
Should we mention can do 'conda install' but 'mamba' recommended like the docs?

This is the only suggestion I have, otherwise looks good! Good work!

Reply:
I missed the conda mention. above. Approved